docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -8,9 +8,13 @@ FROM ubuntu:${ubuntu_version} AS common_base
 # Ubuntu Setup
 ENV TZ=America/Chicago
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+ENV HOME /root
 
 FROM common_base AS apt_deps
-ENV HOME /root
+# set HDF5 path for base_python stage
+ARG build_hdf5="NO"
+ENV HDF5_INSTALL_PATH=$HOME/opt/hdf5/$build_hdf5
```

Comment:
As mentioned earlier, $HOME is not necessary.

Reply:
I think it's necessary because the `/opt` directory is different from the `/root/opt` directory, which is what `$HOME/opt` expands to.

Reply:
We can ignore the extra path /root (not necessary).
In linux we use /opt for 3rd party apps. I think installing hdf5 in /opt is fine. And the same method goes for others.

Reply:
Let's keep `/root` for now since it's been part of our CI strategy for a while - try not to change too many things at once